firebase-firestore/src/main/java/com/google/firebase/firestore/local/IndexManager.java

```diff
@@ -87,14 +87,16 @@
   /** Returns the documents that match the given target based on the provided index. */
   Set<DocumentKey> getDocumentsMatchingTarget(FieldIndex fieldIndex, Target target);
 
-  /** Returns the next collection group to update. */
+  /** Returns the next collection group to update. Returns {@code null} if no group exists. */
   @Nullable
-  String getNextCollectionGroupToUpdate(Timestamp lastUpdateTime);
+  String getNextCollectionGroupToUpdate();
+
+  /** Sets the collection group's latest read time. */
```

Comment:
Update comment to include the fact that it updates all field indexes' index state that match the collection group + include the part where it updates the sequence counter as well